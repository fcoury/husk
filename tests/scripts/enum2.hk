enum Option {
  None,
  Some(String),
}

fn main() {
  let option = Option::Some("Hello".to_string());
  match option {
    Option::None => println!("None"),
    Option::Some(value) => println!("{}", value),
  }
}
