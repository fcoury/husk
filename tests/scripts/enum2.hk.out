Option::Some(Felipe)
Option::None
